### DISCUSSION TITLE
Having issue installing Github Copilot extension

### DISCUSSION THREAD

**Discussion by AyJobee1** [giswqs/geog-312#54] [General]
Dear Professor,
I kept encountering this error when attempt to install Github Copilot extension in VS code. how can I fix it
![Screenshot 2024-09-12 002011](https://github.com/user-attachments/assets/2bdae28f-6ddc-4eea-b509-3c84e75e4751)

**Comment by giswqs** [comment]
You selected the wrong one. It should be the first on the list

**Reply by giswqs** [reply]
If it doesn't work, you can report it to the website shown in the extension page

**Reply by AyJobee1** [reply]
Thank you,
I installed it manually
